components/threads/actions/post-thread/post-thread.mjs

```diff
@@ -3,8 +3,8 @@ import threads from "../../threads.app.mjs";
 export default {
   key: "threads-post-thread",
   name: "Post a Thread",
-  description: "Post a new thread to a specific channel",
-  version: "0.2.0",
+  description: "Post a new thread to a specific channel. [See the Documentation](https://github.com/ThreadsHQ/api-documentation#post-thread).",
```

Comment:
```suggestion
  description: "Post a new thread to a specific channel. [View the documentation](https://github.com/ThreadsHQ/api-documentation#post-thread)."
```

Reply:
Changed 'See the' to 'View the' per Google's style guide

Reply:
"See the documentation" exists everywhere in this repo.